src/Main.cc

```diff
@@ -373,8 +424,9 @@ static int tclAppInit(int& argc,
       // script is done.
       Tcl_StaticPackage(
           interp, "tclreadline", Tclreadline_Init, Tclreadline_SafeInit);
-      if (Tcl_EvalFile(interp, TCLRL_LIBRARY "/tclreadlineInit.tcl")
-          != TCL_OK) {
+
+      std::string path = findPathToTclreadlineInit(interp);
+      if (path.empty() || Tcl_EvalFile(interp, path.c_str()) != TCL_OK) {
```

Comment:
Rather than always taking this path, can we first do what was already there
```
    if (Tcl_EvalFile(interp, TCLRL_LIBRARY "/tclreadlineInit.tcl")
          != TCL_OK) {
```
and treat your solution as a fallback.  It seems less likely to be disruptive to standard use case.

Reply:
Then

Reply:
?

Reply:
fat fingers, hit send early, response corrected.

Reply:
gotit. Fixed.